scripts/lib/upgrade-zulip-stage-2

```diff
@@ -82,6 +87,25 @@ try:
 except (configparser.NoSectionError, configparser.NoOptionError):
     tornado_processes = 1
 
+# postgresql.version is effectively required, but wasn't previously;
```

Comment:
I think we can just s/effectively required/required/`.  In an ideal world, we wouldn't add this unless the system is configured to run `postgres` locally; otherwise, we'll add this on application servers, where it's useless.  I wonder what an appropriately cheap test would be for that; `puppet_classes` are kinda annoying to check.  Checking if `process_fts_updates` is installed would be really nice, but that's wrong for `dockervoyager.pp`.  Hrm.  

Oh, I suppose we can reasonably just check for `/var/lib/postgresql` being present; while that will be wrong on CentOS when we support that, CentOS will also only get the "new system" case anyway.

Reply:
`install` uses `/etc/init.d/postgresql` as its judge, which seems reasonable.

Reply:
Ahh, great, let's do that.

Reply:
This doesn't work because this is *before* we've run puppet, so we may not have postgres installed _yet_, but will get it.

I don't see a way around parsing the puppet classes, since that's the only place that the _intent_ of the install is captured, as opposed to its current state.